ApnsPHP/SharedConfig.php

```diff
@@ -507,12 +507,13 @@ protected function httpInit()
      */
     protected function getJsonWebToken()
     {
-        $key = new Key\LocalFileReference('file://' . $this->providerCertFile);
-        return (string) Configuration::forUnsecuredSigner()->builder()
+        $key = Key\LocalFileReference::file('file://' . $this->providerCertFile);
```

Comment:
```suggestion
        $key = Key\LocalFileReference::file($this->providerCertFile);
```
The file prefix will be trim (https://github.com/lcobucci/jwt/blob/4.2.x/src/Signer/Key/LocalFileReference.php#L28-L30)

Reply:
Ah, thanks! I didn't notice that.